manifests/2.0.1/opensearch-2.0.1.yml

```diff
@@ -52,3 +52,65 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version: notifications
+  - name: anomaly-detection
+    repository: https://github.com/opensearch-project/anomaly-detection.git
+    ref: '2.0'
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version
+  - name: opensearch-observability
+    repository: https://github.com/opensearch-project/observability
```

Comment:
Does this also needs to be a `.git` url like others?

Reply:
I think this shouldn't matter. I check previous few manifest and for `observability` plugin, the repository link is always with `.git` like others. I can add it if you think I have to.

Reply:
Lets keep it uniform and add it.
Thanks!

Reply:
Added. Please help review it now. Thanks.